io.catenax.mp_standard_response/1.0.0/ShopfloorInformationTypes.ttl

```diff
@@ -0,0 +1,88 @@
+##########################################################################################
```

Comment:
as mentioned in the other review. Move this model to its own namespace and folder
also Order wie folgt anlegen
io.catenax.shared.shopfloor_information_types/1.0.0/
und dort die Modelldatie sowie die metadaten rein